app/models/job.rb

```diff
@@ -20,4 +22,10 @@ class Job < ActiveRecord::Base
             presence: true,
             inclusion: { in: STATES }
   validates :author, presence: true
+
+  private
+
+  def set_published_at
+    self.published_at = DateTime.current
```

Comment:
`.touch` is using the method `. current_time_from_proper_timezone `(https://github.com/rails/rails/blob/33e60514aed85b3076f2636d5f1ccfb513aace1c/activerecord/lib/active_record/timestamp.rb#L77) to generate the timestamp. 

By default, the timezone is set to `UTC` and it wasn't change in the config.